contributing_to_docs/doc_guidelines.adoc

```diff
@@ -128,62 +128,41 @@ ID. For example:
 ====
 
-Task modules take the following form:
-
-----
-<module_metadata>
-
-After the module file metadata, include a paragraph explaining why the user must
-perform this task. This should be 1-2 sentences maximum.
-
-If applicable, include any gotchas (things that could trip up the user or cause the task to fail).
-
-== Prerequisites
-
-* A bulleted list of prerequisites that MUST be performed before the user can complete this task.
-Skip if there isn't any related information.
-
-.Procedure
-
-. Step 1 - One command per step.
-
-. Step 2 - One command per step.
-
-. Step N
+== Writing concepts
+A _concept_ contains information to support the tasks that users want to do and
+must not include task information like commands or numbered steps. In most
+cases, create your concepts as individual modules and include them in
+appropriate assemblies.
 
-== Next steps
+Avoid using gerunds in concept titles. "About <concept>"
+is a common concept module title.
 
-You can explain any other tasks that MUST be completed after this task. You can
-skip this if there are none. Do not include xrefs. If the next steps are closely
-related to the task, you might be able to include their modules in the assembly.
+For more information about creating concept modules, see the
+link:https://redhat-documentation.github.io/modular-docs/[Red Hat modular docs reference guide] and the link:https://github.com/redhat-documentation/modular-docs/blob/master/modular-docs-manual/files/TEMPLATE_CONCEPT_concept-explanation.adoc[concept template].
 
-.Related information
+== Writing tasks
+A _task_ contains the steps that users follow to complete a process. A task, also known as a _procedure_, contains ordered steps and explicit commands. In most cases, create your tasks as individual modules and include them in appropriate assemblies.
 
-* A bulleted list of links to related information about this task. Skip if there isn't any related information.
-----
+Use a verb in the task title, such as "Creating".
 
 For more information about writing tasks, see the
-link:https://redhat-documentation.github.io/modular-docs/[Red Hat modular docs reference guide].
+link:https://redhat-documentation.github.io/modular-docs/[Red Hat modular docs reference guide] and the link:https://github.com/redhat-documentation/modular-docs/blob/master/modular-docs-manual/files/TEMPLATE_PROCEDURE_doing-one-procedure.adoc[procedure template].
+
+[NOTE]
+====
+If you include `Prerequistes`, `Next steps`, or `Additional resources` headings in your procedure, use only `==` formatting in assembly files. For module files, use `.Prerequisites`, `.Next steps`, or `.Additional information` syntax to avoid TOC formatting within the module. Do not include xrefs in modules.
```

Comment:
Suggest making this more targeted to modules. I almost read this first as it should use == since that is what is listed first. Maybe simplify to something like:

```
"If the sections are applicable to your module, use the following syntax for these standard headings: ".Prerequisites", ".Next steps", ".Additional resources". Do not use "==" headings for these sections in a module. Remember that modules cannot include any xrefs.
```
^ I don't love my wording attempt, but you get the idea - to include the part for modules first before saying what NOT to do.

Reply:
Thanks, @bergerhoffer! I tried to reword so that it explains how to format if it is a module, while still leaving the possibility that it might be an assembly. But didn't want to over-complicate it too much by adding something like, "assembly files can use `==` formatting. Let me know if it doesn't seem clear enough.

Reply:
Ah, I think I see what you're getting at, that procedures could be directly in an assembly? I would think that all procedures should really be in a module, but perhaps there is a use case for being directly in an assembly. So that's fine to keep as you have it.

Reply:
I feel like I've seen a proc as an assembly but not sure where, and it would certainly be an exception. I've tightened up the wording a bit to favor procedures as modules.

Reply:
:rotating_light: Ancient thread revival thanks to the BLAME button :rotating_light: 

The way this is written still seems to be confusing people into thinking you can't use `.Additional resources` in a module (which, because they often include an `xref`, a lot of times get actually attached right after the module's include at the assembly level, but are not intended to be at the assembly's TOC level). The NOTE further down in the Writing procedures section is more clear on this, but maybe they're just stopping with a CTRL+F on "Additional r..."

Also it still says "Additional information", which I assume is meant to meant "Additional resources" (since the former doesn't appear anywhere in the [mod docs guide](https://redhat-documentation.github.io/modular-docs/)).

Happy to adjust these things myself, but just wanted to make sure I wasn't misunderstanding the original intent here.

Reply:
I agree with @adellape that this needs work. Addressed in https://github.com/openshift/openshift-docs/pull/30125 - thanks!